components/tag-connect/common.stanza

```diff
@@ -0,0 +1,60 @@
+#use-added-syntax(jitx)
+defpackage ocdb/tag-connect/common : 
+  import core
+  import collections
+  import jitx
+  import jitx/commands
+  import ocdb/bundles
+
+public defn make-10-pin-connector-module (component:Instantiable) : 
+  inside pcb-module : 
+    ; the connecting component
+    inst conn: component
+
+    ; the bundles we support out of the module
+    val power-bundle  = power
+    val reset-bundle  = reset
+    val swd-bundle    = swd([SWD-SWO, SWD-TRACESWO]) 
+    val jtag-bundle = jtag()
+
+    port jtag  : jtag-bundle
+    port swd   : swd-bundle
+    port power : power-bundle
+    port reset : reset-bundle
+
+    ; hook up power
+    net (power.vdd, conn.p[1])
+    net (power.gnd, conn.p[3], conn.p[5], conn.p[7], conn.p[9])s
```

Comment:
the s at the end results in an error when trying to use.

Reply:
Fixed!